model/map/src/main/java/org/keycloak/models/map/role/MapRoleAdapter.java

```diff
@@ -144,6 +144,10 @@ public Stream<String> getAttributeStream(String name) {
         return getAttributes().getOrDefault(name, Collections.EMPTY_LIST).stream();
     }
 
+    public RealmModel getRealm() {
```

Comment:
The method could be probably introduced in `RoleModel` as the "other" models also has access to the `RealmModel` as well. 

@hmlnarik I'm not sure whether this can be postponed too much as the `RoleModel` is located in `server-spi` package and I'm not sure if we would be able to change it in future.